test/integration/helpers/inputs.js

```diff
@@ -38,6 +41,40 @@ var findValueInInputs = function(inputs, value) {
           });
 };
 
+// example:
+//   check(element(by.css('input[type="checkbox"]')))
+var check = function(checkboxElem) {
+  // var isChecked = checkboxElem.getAttribute('checked');
+  // if(!isChecked) {
+  //   checkboxElem.click();
+  // }
+  return checkboxElem
+          .isSelected()
+          .then(function(selected) {
+            if(!selected) {
+              return checkboxElem.click();
+            }
+          });
```

Comment:
Most of the codebase uses indentation style:

```js
  return checkboxElem.isSelected().then(function(selected) {
    if(!selected) {
      return checkboxElem.click();
    }
  });
```

I think we should start using that consistently.